website/content/docs/k8s/installation/vault/data-integration/server-tls.mdx

```diff
@@ -42,7 +40,27 @@ which also uses an intermediate signing authority.
 
   -> **Note:** Where `common_name` is comprised of combining `global.datacenter` dot `global.domain`.
 
-### Create Vault Policies for the Server TLS Certificates
+## Overview
+To use an Vault as the Server TLS Certificate Provider on Kubernetes, we will need to modify the steps outlined in the [Data Integration](/docs/k8s/installation/vault/data-integration) section:
+
+- One time set up in Vault
+  1. ~~Store the secret in Vault.~~
```

Comment:
```suggestion
```


Perhaps we can remove this from the list but still mention it below in the case of posterity to previous setup instructions.

Reply:
I found it a looking a little odd that this was striked out.